08.ls_object/lib/path.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+class Path
+  attr_reader :name, :links, :user_name, :group_name, :file_size, :last_update_time, :permission, :type
+
+  PERMISSION_TABLE = { '0' => '---', '1' => '--x', '2' => '-w-', '3' => '-wx', '4' => 'r--', '5' => 'r-x', '6' => 'rw-', '7' => 'rwx' }.freeze
```

Comment:
こんな感じで書いてあげると見やすいと思います。
```suggestion
  PERMISSION_TABLE = {
    '0' => '---',
    '1' => '--x',
    '2' => '-w-',
    '3' => '-wx',
    '4' => 'r--',
    '5' => 'r-x',
    '6' => 'rw-',
    '7' => 'rwx'
  }.freeze
```

Reply:
@summertree128 さんレビューありがとうございます！

こちらのコミットで修正いたしましたのでご確認よろしくお願いします。
[c03cf53](https://github.com/OdenTakashi/ruby-practices/pull/10/commits/c03cf534834233213385e227ce5fd9fbafe3d7f6)